### DISCUSSION TITLE
Ability to add to the current firmware

### DISCUSSION THREAD

**Discussion by jjsektor** [gtxaspec/wz_mini_hacks#575] [General]
Hello gtxaspec,

Thank you for the great work, this project has helped me a lot,
I want to ask, is it possible, to:
1. update the current wyze cam v3 to the latest version 
2. install your hack `factory_t31_ZMC6tiIDQN`
3. after sshing to the device, modify the original firmware rcS or any other file to add an ssh
4. remove your firmware and have ssh option on the latest version?

the whole reason I am asking this is to have ssh and access the "original" firmware filesystem

Thank you so much for yout time :)

**Comment by gtxaspec** [comment]
Yes, in an upcoming release.
